data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/RecordDispatcherImpl.java

```diff
@@ -334,6 +352,11 @@ private Tags getTags(HttpResponse<?> response, ConsumerRecordContext recordConte
   }
 
   private Tags getTags(final ConsumerRecordContext recordContext) {
+    if (recordContext.getRecord().record().value() instanceof InvalidCloudEvent) {
+      return this.resourceContext.getTags().and(
+        Tag.of(Metrics.Tags.EVENT_TYPE, "InvalidCloudEvent")
```

Comment:
nit: can we use a constant?

Reply:
Added in 783e018ea0e585b915a3642bf3dd5a37189fd427